pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532</version>
+    <version>1.565</version>
```

Comment:
The main reason is the change in which Matrix project was moved to a separate plugin.
The addition of the Matrix rerun functionality was tested with this version, and retrospectively - it might was needless